src/main/scala/com/bazaarvoice/sswf/service/WorkflowManagement.scala

```diff
@@ -198,6 +198,34 @@ class WorkflowManagement[SSWFInput, StepEnum <: (Enum[StepEnum] with WorkflowSte
   }
 
   /**
+    * List all open executions for this domain, workflow, and workflowId started after specific time (regardless of version)
+    *
+    * @param from       Start time to search
+    * @param workflowId The particular workflow id to list executions for
+    * @return an unmodifiable list of matching executions
+    */
+  def listOpenExecutions(workflowId: String, from: Date): java.util.List[WorkflowExecutionInfo] = {
+    def innerListOpenExecutions(listOpenWorkflowExecutionsRequest: ListOpenWorkflowExecutionsRequest): java.util.List[WorkflowExecutionInfo] = {
```

Comment:
Since this is exactly the same as `innerListExecutions` I think maybe we should try to combine them, instead of duplicating? For example, `innerListExecutions` can take Options, and then this method can just pass in `None` for closed request?